internal/session/query.go

```diff
@@ -407,6 +407,25 @@ and
 and
 	session_state.start_time < wt_sub_seconds_from_now(@threshold_seconds)
 ;
+`
+	sessionCredentialRewrapQuery = `
+select distinct
+  cred.session_id,
+  cred.key_id,
+  cred.credential,
+  cred.credential_sha256
+from session
+  inner join session_credential cred
+    on cred.session_id = session.public_id
+      and cred.key_id = ?
+where session.project_id = ?
```

Comment:
Re: should this comparison be part of the `where`?
```suggestion
where session.project_id = ? and cred.key_id = ?
```